### PR TITLE
Use the new metadataUrl ability on the client to figure out demensions post transform

### PR DESCRIPTION
**Pull request by snide**
Need to switch back to the latest client once it merges.

### CODE REVIEW

File: utils/metadata.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// Grabs the demensions and other info of a transformed image
+export const fetchMetadata = async (metadataUrl: string) => {
```
Comment (SferaDev) [inline utils/metadata.ts:2]: You can type the response too for more type safety

Comment (snide) [reply utils/metadata.ts]: I pulled a @SferaDev on you and already had this coming! Check it out.

**Review by SferaDev — APPROVED** [review]
We can likely export those interfaces if you want

**Comment by snide** [comment]
> We can likely export those interfaces if you want

It would be helpful and save everyone from writing boilerplate.

**Comment by SferaDev** [comment]
Feel free to open a PR in `client-ts` in the `transformations.ts` file.

**Pull request merged**